cpp/src/parquet/metadata.h

```diff
@@ -210,6 +210,12 @@ class PARQUET_EXPORT RowGroupMetaData {
 
   /// \brief Total byte size of all the uncompressed column data in this row group.
   int64_t total_byte_size() const;
+
+  // The file_offset field that this method exposes is optional. This method
```

Comment:
Please move this below \brief with a new line and use triple slash '///'

Reply:
Sure